### DISCUSSION TITLE
The meaning of PRESSURE in FDS

### DISCUSSION THREAD

**Discussion by mcgratta** [firemodels/fds#13756] [Polls]
In a [recent issue](https://github.com/firemodels/fds/issues/13673), we encountered a few problems related to pressure in compartments resulting from the opening and closing of doors. After fixing a clear bug, we still found unphysical results in the `PRESSURE` field. That is, the FDS output quantity `PRESSURE`, which is actually the perturbation of the background pressure in each sealed compartment, the so-called "pressure zones". Long story short, the background and perturbation pressures fluctuated as doors were opened and closed such that the perturbation alone appeared to make no sense. Only when we looked at the sum of the background and perturbation pressures did things start to make sense. That sum is known as `ABSOLUTE PRESSURE`. It is much like looking at degrees K instead of C. 

The purpose of this informal poll, besides my desire just to test this GitHub feature, is to see whether or not we should change the definition of `PRESSURE` such that it is more like one would expect; that is, what one would measure with a pressure gauge.  Specifically, FDS decomposes the pressure $p$ as follows:
$p(x,y,z,t) = \overline{p}(z,t) + \tilde{p}(x,y,z,t)$
The term $\tilde p$ is the output quantity `PRESSURE`. In cases where the domain is not very tall and there are not sealed compartments, this `PRESSURE` is what one would expect. But in cases like the one cited above, if the two pressure terms change dramatically, the output `PRESSURE` might be confusing. I would like to propose that we change `PRESSURE` to $p - p_\infty$, where $p_\infty$ is the pressure at the "floor", i.e. $z=0$. This parameter can be changed via `P_INF` on the `MISC` line. The reason for subtracting $p_\infty$ is simply to avoid dealing with 5 digit values of pressure in units of Pascal, plus, it is customary to consider the "gauge" pressure. What most people would notice with this change is that now one would see a slight vertical profile in pressure. I find this to be useful when looking at, say, stairwells and vertical shafts.

**Comment by drjfloyd** [comment]
There are multiple types of pressure gauges.  There are absolute pressure gauges, relative pressure gauges, and differential pressure gauges. Another approach is to name the QUANTITY to better reflect what it measures.  So ABSOLUTE PRESSURE would be the barometer type pressure (what gives you density), RELATIVE PRESSURE would substract out some REFERENCE_PRESSURE (generally 1 atm), and DIFFERENTIAL PRESSURE could use XB as the input and give the difference in absolute pressure between the points. Could still have PERTURBATION PRESSURE to just give the component driving flow and ZONE or BACKGROUND PRESSURE to give PBARP. Though for ZONE or BACKGROUND PRESSURE still be the question of using a REFERENCE_PRESSURE or not.

REFERENCE_PRESSURE could be a MISC input to set the default and then also a DEVC to overide the default if one for some reason wanted different reference points for different locations.

**Reply by drjfloyd** [reply]
Could also call relative GAUGE PRESSURE which is how you would see it in something like a Grainger catalog.

**Comment by Er9y714** [comment]
I voted `YES`, while my perspective may be limited to my specific use case, where I mainly use pressure output to validate results, verify boundary conditions, and check if pressure differences across doors exceed maximum thresholds.

If the user has options as @drjfloyd proposed, this already can solve the said issue for users. For common compartment fire simulations, I would use REFERENCE_PRESSURE as a default.
